Avoid autoreconf failure by globbing everything in [] in the macro definition.

Gentoo: http://bugs.gentoo.org/show_bug.cgi?id=230325

--- a/common/gst-feature.m4	2009-07-27 11:29:22.000000000 +0200
+++ b/common/gst-feature.m4	2009-07-27 11:29:28.000000000 +0200
@@ -46,13 +46,13 @@
 dnl GST_PLUGINS_NO will contain those that won't be built
 
 AC_DEFUN([GST_CHECK_FEATURE],
+[dnl
 echo
 AC_MSG_NOTICE(*** checking feature: [$2] ***)
 if test "x[$3]" != "x"
 then
   AC_MSG_NOTICE(*** for plug-ins: [$3] ***)
 fi
-[dnl
 builtin(define, [gst_endisable], ifelse($5, [disabled], [enable], [disable]))dnl
 dnl if it is set to NO, then don't even consider it for building
 NOUSE=
